arch/Kconfig

```diff
@@ -118,6 +118,7 @@ config ARCH_SIM
 	select SERIAL_CONSOLE
 	select SERIAL_IFLOWCONTROL
 	select SCHED_HPWORK
+	select ARCH_HAVE_CPUINFO
```

Comment:
rebase to the last master to remove this change

Reply:
Done.